README.rst

```diff
@@ -258,6 +258,34 @@ For accounts in regions outside of US-West, use :code:`region` parameter to spec
     $dbh = new PDO("snowflake:account=testaccount.us-east-1", "user", "password");
     $dbh = new PDO("snowflake:account=testaccount;region=us-east-1", "user", "password");
 
+Using Key Pair Authentication
```

Comment:
Do you have a separate PR to add the PHP PDO driver to https://docs.snowflake.com/en/user-guide/key-pair-auth.html#supported-snowflake-clients ?

Reply:
No, but I will create one now. Or should I just add it to this PR?

Reply:
SNOW-649074